packages/react-percy-snapshot-render/src/__tests__/render-tests.js

```diff
@@ -17,6 +17,9 @@ jest.mock('redbox-react', () => ({ error }) =>
 );
 /* eslint-enable react/display-name, react/prop-types */
 
+// eslint-disable-next-line no-console
+console.error = jest.fn();
```

Comment:
This test purposefully causes an error to get thrown so we can verify it's getting rendered. React 16 has a new mechanism for catching errors and prints a console error here, but it's not really useful in the context of this test.